packages/main/src/CustomListItem.ts

```diff
@@ -16,26 +16,16 @@ import customListItemCss from "./generated/themes/CustomListItem.css.js";
  *
  * The component accepts arbitrary HTML content to allow full customization.
  *
- * <h3>CSS Shadow Parts</h3>
- *
- * <ui5-link target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/::part">CSS Shadow Parts</ui5-link> allow developers to style elements inside the Shadow DOM.
- * <br>
- * The <code>ui5-li-custom</code> exposes the following CSS Shadow Parts:
- * <ul>
- * <li>native-li - Used to style the main li tag of the list item</li>
- * <li>content - Used to style the content area of the list item</li>
- * <li>detail-button - Used to style the button rendered when the list item is of type detail</li>
- * <li>delete-button - Used to style the button rendered when the list item is in delete mode</li>
- * <li>radio - Used to style the radio button rendered when the list item is in single selection mode</li>
- * <li>checkbox - Used to style the checkbox rendered when the list item is in multiple selection mode</li>
- * </ul>
+ * @csspart native-li - Used to style the main li tag of the list item
+ * @csspart content - Used to style the content area of the list item
+ * @csspart detail-button - Used to style the button rendered when the list item is of type detail
+ * @csspart delete-button - Used to style the button rendered when the list item is in delete mode
+ * @csspart radio - Used to style the radio button rendered when the list item is in single selection mode
+ * @csspart checkbox - Used to style the checkbox rendered when the list item is in multiple selection mode
  *
  * @constructor
```

Comment:
- Make the CustomListItem class to explicitly implements IListItem in TypeScript
- Remove `@type` tag
- Move default slot comment to class comment (check how to describe unnamed slots)

Reply:
I don't see any slots in this file.